app/controllers/help_controller.rb

```diff
@@ -0,0 +1,50 @@
+#  Copyright (c) 2019, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+class HelpController < ApplicationController
+
+  POWER_USER_ROLE_TYPES = %w(Group::Bund::PowerUser Group::Kantonalverband::PowerUser
+                             Group::Region::PowerUser Group::Abteilung::PowerUser).freeze
+
+  skip_authorization_check only: [:index]
+
+  def index
+    @groups = groups_of_current_user.map do |group|
+      [group, power_users_for_group_or_above(group)]
+    end
+  end
+
+  private
+
+  def groups_of_current_user
+    current_user.roles.map do |role|
+      groups.find { |group| group.id == role.group_id }
+    end.uniq
+  end
```

Comment:
`Person` hat auch direkt die `groups` verfügbar (als `has_many :through`), das würde ich eher verwenden. Damit fällt dann auch die Method `groups` weiter unten raus. Vermutlich kann man das dann auch gleich "inline" oben in action machen. Das `uniq` könnte dennoch nötig sein, wenn man in der gleichen Gruppe mehrere Rollen hat.